pkg/apis/postgres-operator.crunchydata.com/v1beta1/postgrescluster_types.go

```diff
@@ -361,6 +365,17 @@ type PostgresInstanceSetSpec struct {
 	WALVolumeClaimSpec *corev1.PersistentVolumeClaimSpec `json:"walVolumeClaimSpec,omitempty"`
 }
 
+// InstanceSidecars defines PostgreSQL instance sidecar containers
+type InstanceSidecars struct {
+	// Defines the configuration for the replica cert copy sidecar container
+	// +optional
+	ReplicaCertCopy *Sidecar `json:"replicacertcopy,omitempty"`
```

Comment:
:wrench: Thinking we should camelCase here (e.g. `replicaCertCopy`) for consistency with other areas of the spec.

Reply:
:+1: updated.